### PR TITLE
v1.13.2 geoips_tropics - VERSION RELEASE - distro statements, alg data range

### PR DESCRIPTION
**Pull request by mindyls**
# Related Issues
required to close NRLMMD-GEOIPS/geoips#740

# Reviewer Instructions
* Please confirm this PR is set up to merge from v1.14.1-version-release branch into main branch
* Review "Files changed" tab
* Confirm appropriate testing was completed for these updates (you may perform tests yourself, or review output included from another reviewer/assignee)

# Summary
Changes for version 1.13.2, uploaded with
1.14.1 update on repo geoips_tropics.
See release note updates in 'Files changed' tab

See NRLMMD-GEOIPS/geoips#740 for additional repositories included in this update.

# Testing Instructions
See NRLMMD-GEOIPS/geoips#740 for testing instructions.

# Output
See NRLMMD-GEOIPS/geoips#740 for test output.

# Post Merge Steps
Once this PR has been merged, v1.13.2
will be tagged/released on the geoips_tropics main branch.

### CODE REVIEW

File: docs/source/releases/index.rst [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
- | # # # This program is free software: you can redistribute it and/or modify it under
- | # # # the terms of the NRLMMD License included with this program. This program is
- | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
- | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
- | # # # for more details. If you did not receive the license, for more information see:
- | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+.. dropdown:: Distribution Statement
+
+ | # # # This source code is protected under the license referenced at
+ | # # # https://github.com/NRLMMD-GEOIPS.
```
Comment (mindyls) [inline docs/source/releases/index.rst:4]: Another one we should probably remove index.rst, and update everything for ``cp -rp ../geoips_ci/sync/. .`` before merging.  This was tagged before updating CI

Comment (jsolbrig) [reply docs/source/releases/index.rst:4]: Does this need to be done before merging this one?

**Review by jsolbrig — DISMISSED** [review]

File: docs/source/releases/index.rst (hunk 1 above)
Comment (mindyls) [reply docs/source/releases/index.rst]: Yes, done.

**Review by jsolbrig — APPROVED** [review]

**Comment by mindyls** [comment]
Ready  to merge once we have 2 approvals and all conversations are resolved.

The one failing action is release note check - these just have header updates, and we should never have modifications to RST release note files again since we now auto-generate RST release notes.  We should merge with this one remaining failing check since these are expected changes to existing RST release notes.

image

image

**Review by cpcamacho — APPROVED** [review]

**Pull request merged**